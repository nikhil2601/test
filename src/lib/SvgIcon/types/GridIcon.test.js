import React from 'react';
import { shallow } from 'enzyme';

import GridIcon from './GridIcon';

describe('<GridIcon />', () => {
    const wrapper = shallow(<GridIcon />);

    it('should render correctly', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
