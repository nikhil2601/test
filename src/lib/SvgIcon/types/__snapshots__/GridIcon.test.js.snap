// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridIcon /> should render correctly 1`] = `
<SvgIcon
  color="inherit"
  height="1rem"
  margin={null}
  opacity={null}
  padding={null}
  titleAccess={null}
  viewBox="0 0 24 24"
  width="1rem"
>
  <path
    d="M16,20H20V16H16M16,14H20V10H16M10,8H14V4H10M16,8H20V4H16M10,14H14V10H10M4,14H8V10H4M4,20H8V16H4M10,20H14V16H10M4,8H8V4H4V8Z"
  />
</SvgIcon>
`;
